client/README.md

```
@@ -1,44 +1,51 @@
+- **Workspace Symbols**: Search for symbols across the entire workspace
+- **[Data Browser](https://github.com/jbearak/sight/blob/main/docs/data-browser.md)**: Open `.dta` files directly in VS Code, or call `vview` from Stata to send the current dataset to the editor — features a virtualized grid with column resizing/hiding and value labels
+- **[Log Viewer](https://github.com/jbearak/sight/blob/main/docs/log-viewer.md)**: Render Stata `.smcl` log files with formatted output directly in VS Code
+- **[Help Viewer](https://github.com/jbearak/sight/blob/main/docs/help-viewer.md)**: Read Stata `.sthlp` help files directly in VS Code with clickable help-topic links
+
+## Screenshots
+
+<details>
+<summary>Real-time diagnostics</summary>
 
-Context-aware completions for commands, options, macros, and variables.
+<img width="683" height="390" src="https://github.com/jbearak/sight/blob/main/examples/undefined_local.png?raw=true"/>
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)